### PR TITLE
V2 frame

### PR DESCRIPTION
**Pull request by muke1908**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,118 +1,43 @@
-# Entify Web SDK
+# web-sdk-frame
 
-The Entify SDK provides you with a building block to be included in your JavaScript application. It can be used to quickly and conveniently integrate our legal entity screening solution to your platform or webpage.
+This is a frame that is used in entify web sdk v2
```
Comment (nelesergejeva) [inline README.md]: Would be nice to explain with a few sentences how and why the frame is used.

Comment (nelesergejeva) [reply README.md]: Please also capitalise SDK everywhere and end the readme sentences with a .

File: README.md [hunk 2]
```diff
@@ -1,118 +1,42 @@
...
-## Setting up the SDK
+This is a static website containing the SDK flow. The Entify SDK internally loads this website through **iframe**.  
+Implementation [Demo](https://web-sdk-v2.herokuapp.com/)
 
-Before you start integrating with the Entify SDK, make sure that you:
+### Local development:
 
-- have requested an API token and received your API key. If you haven't, please send email to support@entifyme.com to get one.
-- have subscribed to the [Entify webhooks](https://developers.kyc-pass.com/#webhooks).
+The website is built on preact. To initiate the sdk flow, you have to provide a valid token in the url param.
```
Comment (nelesergejeva) [inline README.md]: url => URL

File: README.md [hunk 3]
```diff
@@ -1,118 +1,42 @@
...
-    window.entify.mount({
-      ...
-    })
-  })
-</script>
+npm run dev
 ```
 
-The `mount` function accepts an object with the following 4 properties:
+In dev mode, it'll open in browser. Append a valid token in the url (`/?token=<token>`) to proceed further.
```
Comment (nelesergejeva) [inline README.md]: url => URL

File: README.md (hunk 1 above)
Comment (muke1908) [reply README.md]: I have added at the bottom of the readme.

**Pull request merged**